scripts/scripts.js

```diff
@@ -125,14 +200,15 @@ export function loadSidebar(element) {
 async function loadLazy(doc) {
   const main = doc.querySelector('main');
   await loadBlocks(main);
+  await replacePricePlaceHolders([main]);
 
   const { hash } = window.location;
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
-  loadHeader(doc.querySelector('header'));
-  loadSidebar(doc.querySelector('main'));
-  loadFooter(doc.querySelector('footer'));
+  await loadHeader(doc.querySelector('header'));
```

Comment:
I think we don't want to use an await here

Reply:
My understanding is that since that function returns `  return loadBlock(headerBlock);` which returns a promise, we must handle it somewhere.